desktop/apps/auction/components/Layout.js

```diff
@@ -1,17 +1,23 @@
+import ArtworkBrowser from 'desktop/apps/auction/components/artwork_browser/ArtworkBrowser'
 import AuctionBlock from 'desktop/components/react/auction_block/auction_block'
+import AuctionInfoContainer from 'desktop/apps/auction/components/layout/auction_info'
 import Banner from 'desktop/apps/auction/components/layout/Banner'
-import ArtworkBrowser from 'desktop/apps/auction/components/artwork_browser/ArtworkBrowser'
 import Footer from 'desktop/apps/auction/components/layout/Footer'
-import AuctionInfoContainer from 'desktop/apps/auction/components/layout/auction_info'
+import GridArtwork from 'desktop/apps/auction/components/artwork_browser/main/artwork/GridArtwork'
 import MyActiveBids from 'desktop/apps/auction/components/layout/active_bids/MyActiveBids'
 import PropTypes from 'prop-types'
 import React from 'react'
-import WorksByFollowedArtists from 'desktop/apps/auction/components/artwork_browser/sidebar/WorksByFollowedArtists'
+import get from 'lodash.get'
+import { Artwork } from '@artsy/reaction-force/dist/Components/Artwork'
+import { ArtworkRail } from './layout/artwork_rail/ArtworkRail'
+import { RelayStubProvider } from 'desktop/components/react/RelayStubProvider'
 import { connect } from 'react-redux'
 
 function Layout (props) {
   const {
     associatedSale,
+    buyNowSaleArtworks,
```

Comment:
We may want to change the naming here-- I know that it's called "buy now" on the UI but internally we refer to an auction work that can _also_ be bought through e-commerce as a "buy now" work, and I can see this getting confusing. Can we call it something like `buyableSaleArtworks` or `promotedSaleArtworks` instead? (there are probably other names too 😄 ).